src/root/useRestoredReduxState.ts

```diff
@@ -33,6 +34,9 @@ const useRestoredReduxState = () => {
     // Species Page
     dispatch(restoreSpeciesPageUI());
 
+    // Genome browser
+    dispatch(loadBrowserGeneralState());
```

Comment:
where are we restoring previously viewed objects for the genome browser, by the way?

Reply:
It's done here: https://github.com/Ensembl/ensembl-client/pull/654/files#diff-439f80b216a7f5864a239d9008365d9557be94edae7341a3056bb180fbdb0639R370